pkg/util/util.go

```diff
@@ -161,6 +161,10 @@ func NewIntOrStringFromString(val string) IntOrString {
 	return IntOrString{Kind: IntstrString, StrVal: val}
 }
 
+func (intstr *IntOrString) IntValue() int {
+	return int(intstr.IntVal)
```

Comment:
Call it everywhere, you mean?  Happy to.

On Tue, Nov 17, 2015 at 4:24 PM, Daniel Smith <notifications@github.com>
wrote:

> In pkg/util/util.go
> <https://github.com/kubernetes/kubernetes/pull/17353#discussion_r45123105>
> :
>
> > @@ -161,6 +161,10 @@ func NewIntOrStringFromString(val string) IntOrString {
> >  	return IntOrString{Kind: IntstrString, StrVal: val}
> >  }
> >
> > +func (intstr *IntOrString) IntValue() int {
> > +	return int(intstr.IntVal)
>
> Oh, this function already exists. Can we call it instead of casting?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/17353/files#r45123105>.
>